tests/improved-sanity-test/main.yml

```diff
@@ -366,6 +366,47 @@
       tags:
         - docker_rm_container_again
 
+    # Configure container_manage_cgroup boolean
+    # See - https://bugzilla.redhat.com/show_bug.cgi?id=1510139
+    #     - https://bugzilla.redhat.com/show_bug.cgi?id=1554881
+    - when: ansible_distribution == "RedHat" or
+            ansible_distribution == "Fedora"
+      role: selinux_boolean
+      g_seboolean: "container_manage_cgroup"
+      tags:
+        - container_manage_cgroup_bool
+
+    # TEST
+    # Build non-priviledged systemd httpd container
+    - role: docker_build
+      db_src: "roles/docker_build/files/{{ g_osname }}/systemd_httpd/"
+      db_image_name: "{{ g_osname }}_systemd_httpd"
+      tags:
+        - docker_build_systemd
+
+    # TEST
+    # Run non-privileged systemd httpd container
+    - role: docker_run
+      dr_image_name: "{{ g_osname }}_systemd_httpd"
+      dr_run_options: "-d -p 80:80"
+      tags:
+        - docker_run_systemd
+
+    # TEST
+    # Verify that port 80 is open and the URL is accessible
+    - role: check_open_port
+      cop_port: "80"
+      cop_url: "http://localhost:80/"
+      tags:
+        - check_open_port_systemd
+
+    # TEST
+    # Remove the non-privileged systemd httpd container
+    - role: docker_rm_container
+      drc_container_name: "{{ g_osname }}_systemd_httpd"
+      tags:
+        - docker_rm_container_systemd
+
```

Comment:
I just had a thought while reviewing the code.  Since this is actually a group of roles that test the non-priv systemd container, can we add an additional tag on each of them to group them together (for example adding a tag of `non-priv-systemd-test` to each of these roles)? If there is a bug causing a failure, we can just say `skip-tags non-priv-systemd-test`.  I think this can be done in other parts of the improved sanity test also.  What do you think?

Reply:
Extra tags is easy!  

(I actually had a similar thought and considered making a 'meta' role for this specific test case that just called out to individual roles.  I kind of like having each role available in the test explicitly, though, as it is that much clearer to understand what is happening.  We already have roles with true `meta` requirements, which is just additional layers of the onion to peel back when debugging)

Reply:
I also like having each role available at the test level so its easier to figure out what's happening when there are failures.  Having tags is like extra metadata to see those roles are grouped.  Nice work on this PR!